src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -87,6 +87,10 @@ internal static class FeatureOnOffOptions
             nameof(FeatureOnOffOptions), nameof(UseEnhancedColors), defaultValue: 1,
             storageLocations: new RoamingProfileStorageLocation("WindowManagement.Options.UseEnhancedColorsForManagedLanguages"));
 
+        public static readonly Option<int> UseNullableReferenceTypeAnalysis = new Option<int>(
```

Comment:
If we're doing a 0/1/-1 thing, can we add a comment accordingly?

Reply:
Sure, I wasn't sure if that was some archaic known for int options or what.

Reply:
Nah, it's a new thing we've been doing for ints controlled by flights. We really should just get an enum.